api/lib/domain/models/TargetProfile.js

```diff
@@ -53,7 +53,7 @@ class TargetProfile {
   }
 
   getSkillNames() {
-    return this.skills.map((skill) => skill.name);
+    return this.skills.map((skill) => skill.name).sort();
```

Comment:
je comprends pas le lien de cette ligne modifier avec le nom du commit "Improve unit test of usecase that writes CSV"

Reply:
Moui. En fait, tout du long, j'étais gênée de voir que pour construire le CSV on ne faisait aucun tri sur les compétences ou les acquis. Du coup, un téléchargement après l'autre ne garanti pas l'ordre des colonnes et je trouve ça bizarre.
Je suis d'accord avec toi, la modif je peux la sortir de ce commit.
En tout cas je veux bien qu'on en discute (a posteriori, peut-être pas dans cette PR du coup)

Reply:
Ok je l'ai viré, on verra une autre fois pour l'ordre des colonnes